recipes/antismash/meta.yaml

```diff
@@ -37,31 +33,31 @@ requirements:
     - glimmer ==3.02
     - glimmerhmm ==3.0.4
     - helperlibs
-    - hmmer
+    - hmmer ==3.1b2
     - hmmer2
     - openjdk
     - libxml2 ==2.9.4
     - mafft
     - matplotlib
-    - meme ==4.11.2
+    - meme ==4.12.0
     - muscle
     - networkx
     - numpy
     - pandas
     - perl-bioperl
-    - perl-pathtools
     - prodigal
     - pyexcelerator
     - pyquery ==1.2.9
     - pysvg
     - scipy
-    - scikit-learn ==0.18.2
+    - scikit-learn
```

Comment:
RODEO comes with scikit-learn models trained and pickled with that version of scikit-learn, IIRC. Not sure how those models load back into the current 0.19 series.